core/server/data/migrations/versions/4.0/10-add-members-status-events-table.js

```diff
@@ -0,0 +1,8 @@
+const {addTable} = require('../../utils');
+
+module.exports = addTable('members_status_events', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    member_id: {type: 'string', maxlength: 24, nullable: false, references: 'members.id', cascadeDelete: true},
+    status: {type: 'string', maxlength: 50, nullable: false},
```

Comment:
As above - this is the only table that doesn't store both old and new values. Should we have `from_status` and `to_status` here?

Reply:
Sure thing, will add this

Reply:
I'm wondering actually - do what's the usecase for this?

Reply:
As discussed, this isn't necessary to store